README.md

```diff
@@ -41,6 +41,36 @@ To install it, run:
 go get gopkg.in/src-d/go-mysql-server.v0
 ```
 
+## Regular expression engine
+
+The **go-mysql-server** uses [oniguruma](github.com/kkos/oniguruma) as a default regular expression engine, so it's required to install the oniguruma first.
+For Linux or OSX the easiest way is by command:
+```sh
+make oniguruma # on linux you may need sudo
+```
+
+If you want to use regex engine from go's standard library, you have to compile **go-mysql-server** with `--tags go` argument.
```

Comment:
shall we change the tag to something more specific, like golang_regex_engine or something like that?

Reply:
there are a lot of possibilities that another library is using the same `go` tag

Reply:
if this is a risk, so lets prefix it with `mysql_`

Reply:
In my opinion, `mysql_go` is too broad, but maybe is the go way ;)